runtimes/js/encore.dev/internal/runtime/mod.ts

```diff
@@ -5,5 +5,5 @@ export * from "./napi/napi.cjs";
 const testMode = process.env.NODE_ENV === "test";
 
 export const RT = new Runtime({
-  testMode,
+  testMode
```

Comment:
Could we register the type constructor here instead? Then we don't need to worry about the case where the constructor hasn't been registered?

Reply:
yes! I've added it the runtime constructor now